Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3885,6 +3888,24 @@ def gen2022HiMix(fragment,howMuch):
                          '--datatier':'NANOAODSIM',
                          '--eventcontent':'NANOEDMAODSIM',
                          '--filein':'file:step3_inMINIAODSIM.root'}
+
+steps['HcalNano']={'-s':'RAW2DIGI,RECO,USER:DPGAnalysicalAOD/hcalNano_cff.hcalNanoTask',
```

Comment:
Hi @DryRun 
Typo here?

Reply:
Before you used
`USER:DPGAnalysis/HcalNanoAOD/hcalNano_cff.hcalNanoTask'`

Reply:
Indeed, this is was a typo :( Fixed it.